src/Components/Home/Styles/mid.css

```diff
@@ -0,0 +1,118 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body{
+   font-family: 'Times New Roman', Times, serif;
+  }
+  
+
+  .mid-container{
+    background-image: linear-gradient(black, purple);
+    height: 100vh;
+    width: 100%;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 1rem;
+  }
+  
+  .mid-header{
+    width: 80%;
+    text-align: center;
+    
+    padding: 1rem;
+    font-size: 1.5rem;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  }
+  .pdt-cards{
+    width: 80%;
+    margin-top:5rem;
+    display: flex;
+    gap: 1rem;
+    background: transparent;
+    flex-direction: row;
+    overflow-y:hidden;
+  }
+
+  
+  
+  .bottom-container{
+    background-image:linear-gradient(purple,#001E95);
+    width: 100%;
+    height: 45rem;
+    color: white;
+    display: flex;
```

Comment:
Since you're using flex at many places create a flex className and add it wherever required